moreComfortable.js

```diff
@@ -0,0 +1,114 @@
+//two players will take turns to play
+//when the player clicks submit, game rolls 2 dice and show the values
+//player then can pick the order of dice they want ie: 36 or 63
+//after both players have rolled and chosen the dice order, the player with the highest combined number wins.
+//have to store the values of the chosen combination
+var player1Choice = [0];
+var player2Choice = [0];
+
+//store dice values
+var diceValues = [];
+//since there will be two players, multiple program states needed
+var currentGameMode = "waiting for player 1 input";
+//rolling dice function
+var numberGenerator = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var randomNumber = randomInteger + 1;
+  return randomNumber;
+};
+//we can use loops to store the dice roll values into the array
+var dicePlayer1 = function () {
```

Comment:
The convention is for functions to start with a verb where possible. So instead of dicePlayer1() a better name would be generatePlayer1Dice(). This also makes it easier for us to differentiate functions from variables which reduces the chances of human error when writing code.